client/src/modules/home/index.tsx

```diff
@@ -45,18 +48,28 @@ const Welcome = ({ user }: { user: User }) => {
 };
 const Home = () => {
   const [hasMore, setHasMore] = useState(true);
-  const { loading, error, data, fetchMore } = useHomeQuery({
-    variables: { offset: 0, limit: 2 },
+  const {
+    loading: chapterLoading,
+    error: chapterError,
+    data: chapterData,
+  } = useChaptersQuery();
+
+  const { loading, error, data, fetchMore } = usePaginatedEventsWithTotalQuery({
+    variables: { offset: 0, limit: eventCards, showFiltered: true },
   });
+
   const { user } = useUser();
 
   const toast = useToast();
   const onLoadMore = async () => {
     try {
       const res = await fetchMore({
-        variables: { offset: data?.paginatedEvents.length },
+        variables: { offset: data?.paginatedEventsWithTotal.length },
       });
-      setHasMore(res.data.paginatedEvents.length > 0);
+      const lookIfDataExist =
+        res.data.paginatedEventsWithTotal.length ===
+        data?.paginatedEventsWithTotal.length;
+      setHasMore(lookIfDataExist);
```

Comment:
If the total is odd, it will hide the odd number, although people won't know, it is still a bug, but I have no idea how to fix it.